### DISCUSSION TITLE
Static HTML rendering

### DISCUSSION THREAD

**Discussion by jandusek** [nanojsx/nano#140] [Help / Questions] [answered]
I'd love to find out more about static HTML rendering mentioned in the README:

> **Pre-Rendering**
> Renders your app to static html if you want.
> (This is possible, but requires some knowledge.
> I plan to make a tutorial soon.)

Is this already described somewhere by any chance?

My use case is a purely server-rendered website, with no need for hydration or any client-side DOM changes. Can nanojsx do the server-side render during the build time (say from within gulp)?

**Comment by yandeu** [comment, marked as answer]
Ah I see. Depending on the server you use (eg express) you can simply write a script that crawls all your GET routes and writes them to a file.

This is not something nano-jsx provides.
